assets/gofish/lang/zh_tw.json

```
@@ -0,0 +1,132 @@
+{
+  "itemGroup.gofish.group": "釣魚時光",
+
+  "gofish.crate_tooltip": "Shift + 右鍵打開",

```

```suggestion
  "gofish.crate_tooltip": "Shift + 點擊右鍵開啟",
```
為了翻譯一致性，``Open` 習慣上是譯作 ``開啟``